gencert-private.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+# generate configuration file
+
+openssl_conf=$(mktemp)
+
+cat > "${openssl_conf}" <<EOF
+oid_section=spid_oids
+
+[ req ]
+default_bits=2048
+default_md=sha256
+distinguished_name=dn
+encrypt_key=no
+prompt=no
+req_extensions=req_ext
+
+[ spid_oids ]
+agid=1.3.76.16
+spid-privatesector-SP=1.3.76.16.4.3.1
+spid-publicsector-SP=1.3.76.16.4.2.1
+uri=2.5.4.83
+
+[ dn ]
+commonName=${ENTITY_ID}
+countryName=IT
+localityName=${LOCALITY_NAME}
+stateOrProvinceName=${PROVINCIE}
```

Comment:
I'm pretty sure `stateOrProvincceName` is not allowed. Futhermore, take a look at the `PROVINCIE` typo.